tests/IntegrationTests/EventsBuilderTest.php

```diff
@@ -0,0 +1,156 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\IntegrationTests;
+
+use Lmc\Matej\Exception\RequestException;
+use Lmc\Matej\Model\Command\Interaction;
+use Lmc\Matej\Model\Command\ItemProperty;
+use Lmc\Matej\Model\Command\UserMerge;
+use Lmc\Matej\RequestBuilder\EventsRequestBuilder;
+
+/**
+ * @covers \Lmc\Matej\RequestBuilder\EventsRequestBuilder
+ */
+class EventsBuilderTest extends IntegrationTestCase
+{
+    /** @test */
+    public function shouldExecuteInteractionCommandOnly(): void
+    {
+        $builder = $this->createMatejInstance()->request()->events();
+        $this->appendInteractionCommands($builder);
+        $response = $builder->send();
+
+        $this->assertResponseCommandStatuses($response, ...$this->generateOkStatuses(1));
+    }
+
+    /** @test */
+    public function shouldExecuteUserMergeCommandOnly(): void
+    {
+        $builder = $this->createMatejInstance()->request()->events();
+        $this->appendUserMergeCommands($builder);
+        $response = $builder->send();
+
+        $this->assertResponseCommandStatuses($response, ...$this->generateOkStatuses(1));
+    }
+
+    /** @test */
+    public function shouldExecuteItemPropertyCommandOnly(): void
+    {
+        $builder = $this->createMatejInstance()->request()->events();
+        $this->appendItemPropertyCommands($builder);
+        $response = $builder->send();
+
+        $this->assertResponseCommandStatuses($response, ...$this->generateOkStatuses(1));
+    }
+
+    /** @test */
+    public function shouldExecuteInteractionAndUserMergeCommandsOnly(): void
+    {
+        $builder = $this->createMatejInstance()->request()->events();
+        $this->appendInteractionCommands($builder);
+        $this->appendUserMergeCommands($builder);
+        $response = $builder->send();
+
+        $this->assertResponseCommandStatuses($response, ...$this->generateOkStatuses(2));
+    }
+
+    /** @test */
+    public function shouldExecuteInteractionAndItemPropertyCommandsOnly(): void
+    {
+        $builder = $this->createMatejInstance()->request()->events();
+        $this->appendInteractionCommands($builder);
+        $this->appendItemPropertyCommands($builder);
+        $response = $builder->send();
+
+        $this->assertResponseCommandStatuses($response, ...$this->generateOkStatuses(2));
+    }
+
+    /** @test */
+    public function shouldExecuteUserMergeAndItemPropertyCommandsOnly(): void
```

Comment:
There are many command combinations. IMO we should test only these with real use case.

`[merge], [recommendation]`

`[interaction], [recommendation]`